x/feerefunder/keeper/keeper_test.go

```diff
@@ -0,0 +1,86 @@
+package keeper_test
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	testutil_keeper "github.com/neutron-org/neutron/testutil/keeper"
+	"github.com/pkg/errors"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/neutron-org/neutron/x/feerefunder/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestKeeperCheckFees(t *testing.T) {
+	k, ctx := testutil_keeper.FeeKeeper(t)
+
+	k.SetParams(ctx, types.Params{
+		MinFee: types.Fee{
+			RecvFee:    nil,
+			AckFee:     sdk.NewCoins(sdk.NewCoin("denom1", sdk.NewInt(100)), sdk.NewCoin("denom2", sdk.NewInt(100))),
+			TimeoutFee: sdk.NewCoins(sdk.NewCoin("denom1", sdk.NewInt(100)), sdk.NewCoin("denom2", sdk.NewInt(100))),
+		},
+	})
+
+	for _, tc := range []struct {
+		desc  string
+		fees  *types.Fee
+		valid bool
+	}{
+		{
+			desc: "single proper denom but insufficient",
```

Comment:
also: please stick with the naming pattern: SingleProperDenomButInsufficient

Reply:
fixed